client/src/pages/Profile.tsx

```
@@ -84,496 +101,486 @@ interface Language {
+										className='overflow-x-auto'
+										dangerouslySetInnerHTML={{ __html: githubStreakSvg }}
+									/>
+								) : (
+									<Skeleton className='h-48 w-full' />
+								)}
+
+								<h2 className='text-xl font-bold'>LeetCode Stats</h2>
+								{leetcodeSvg ? (
+									<div
+										className='overflow-x-auto'
+										dangerouslySetInnerHTML={{ __html: leetcodeSvg }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)